docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash --login
```

Comment:
Perhaps we don't need to force the login shell if we're going to manually `source /etc/profile` below? (though I'd assume we'd want to bring lines 9-15 above the `Using python...` echo in that case).

Reply:
I added the `source /etc/profile` after doing the `--login` bit, so yeah, that may no longer be necessary.  Let me try pulling that out and see if/how that changes the environment.

Reply:
Confirmed.  Removed the `--login`, which always felt a little weird.

Reply:
Actually, the `--login` _is_ needed.  Without this, the `source /etc/profile` fails.  Unfortunately, it does not give an error message.  I'd like to understand why, but I don't want to hold up the PR any longer, so I just put the `--login` back in for now.